xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-ui/src/main/java/org/xwiki/test/ui/po/BasePage.java

```diff
@@ -118,6 +126,10 @@ public BasePage()
     {
         super();
         waitUntilPageIsReady();
+
+        WcagContext wcagContext = getUtil().getWcagContext();
```

Comment:
missing comment and should have a check, there'esno need to gather wcag context if wcag is not enabled

Reply:
On this basePage, from what I know, we need to check wcagContext to see if wcag is enabled.